models/shopify_graphql.yml

```diff
@@ -0,0 +1,1202 @@
+      - name: line_item_count_in_month
+        description: Number of line items purchased in the `date_month`
+      - name: line_item_count_lifetime
+        description: Number of line items purchased up until and including this `date_month`.
+      - name: order_count_in_month
+        description: Number of orders purchased in the `date_month`
+      - name: order_count_lifetime
+        description: Number of orders purchased up until and including this `date_month`.
+      - name: source_relation
+        description: The schema or database this record came from, if you are unioning multiple connectors. Null if not.
+      - name: total_price_in_month
+        description: Total amount (in shop currency) purchased in the `date_month`
+      - name: total_price_lifetime
+        description: Total amount (in shop currency) up until and including this `date_month`.
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+
+  - name: shopify_gql__customer_email_cohorts
+    description: Each record represents a customer's (on the EMAIL grain) performance in a calendar month.
+    columns:
+      - name: cohort_month
+        description: The month the cohort belongs to, i.e the first month the customer had an order.
+      - name: cohort_month_number
+        description: The 'number' of the `date_month` of the record, i.e. how many months from their start month this cohort occurred
+      - name: customer_cohort_id
+        description: Unique key representing a customer in a given month. Hashed on 'date_month','email', and 'source_relation'.
+        tests:
+          - unique
+          - not_null
+      - name: email
+        description: The ID of the related customer.
+      - name: date_month
+        description: The calendar month the customer stats relate to.
+      - name: first_order_timestamp
+        description: The timestamp of the customer's first order.
+      - name: line_item_count_in_month
+        description: Number of line items purchased in the `date_month`
+      - name: line_item_count_lifetime
+        description: Number of line items purchased up until and including this `date_month`.
+      - name: order_count_in_month
+        description: Number of orders purchased in the `date_month`
+      - name: order_count_lifetime
+        description: Number of orders purchased up until and including this `date_month`.
+      - name: source_relation
+        description: The schema or database this record came from, if you are unioning multiple connectors. Null if not.
+      - name: total_price_in_month
+        description: Total amount (in shop currency) purchased in the `date_month`
+      - name: total_price_lifetime
+        description: Total amount (in shop currency) up until and including this `date_month`.
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+        
+  - name: shopify_gql__orders
```

Comment:
I noticed the `shipping_costs` field is not documented. Should it be?

Reply:
I just ended up removing `shipping_cost` since it was another duplicative field created for the sake of matching a field name in REST